ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -177,6 +178,7 @@
       @type='password'
       name='client_certificate_key'
       disabled={{form.disabled}}
+      @hasVisibilityToggle={{false}}
```

Comment:
Non-blocking:
Just picking this field as an example but maybe we should audit which fields we are using `@type='password'` with? Looking at the HDS docs, this might be a use case for [masked input](https://helios.hashicorp.design/components/form/masked-input#masked-input-vs-password-input). I will add an agenda item to our tech time.

Reply:
good point. I thought about this as well, and definitely worth a discussion.